test/transactions/3_castVotes.js

```diff
@@ -164,4 +164,104 @@ describe('#castVotes transaction', () => {
 				.and.be.hexString();
 		});
 	});
+
+	describe('when the cast vote transaction is created with the votes and unvotes', () => {
+		beforeEach(() => {
+			castVotesTransaction = castVotes({
+				passphrase,
+				votes: votePublicKeys,
+				unvotes: unvotePublicKeys,
+			});
+		});
+
+		it('the transaction should have the votes object', () => {
```

Comment:
I'm not sure we need this if we have the test below it, but if we do isn't it an array? There's a should.js assertion for that.